### PR TITLE
Fix/dont install make.exe

### PR DESCRIPTION
**Pull request by gares**
Fixes  coq/coq#5987

### CODE REVIEW

**Comment by maximedenes** [comment]
Could you change the commit title to mention the issue? Thanks!

I post a question on the issue thread (is the build button still usable after this PR?)

**Comment by maximedenes** [comment]
@gares ping

**Comment by Zimmi48** [comment]
You shouldn't merge this without testing and you can't test it until #5998 is fixed, right?

**Comment by Zimmi48** [comment]
It seems to me that this PR has been largely ignored since it was created, in part due to the `needs: independent fix` label but also because the question asked by @maximedenes has not been addressed. Thus, I'm adding `needs: discussion` and re-targeting to 8.7.2.

File: dev/build/windows/makecoq_mingw.sh [hunk 1]
```diff
@@ -1275,7 +1275,6 @@ function get_cygwin_mingw_sources {
 
 function make_coq_installer {
   make_coq
-  make_mingw_make
```
Comment (maximedenes) [inline dev/build/windows/makecoq_mingw.sh]: Didn't you forget to remove dead code in the script? In fact, that part broke recently.

**Comment by Zimmi48** [comment]
I don't personally have an opinion on this issue / PR and it seems that it should be discussed primarily between @gares and @MSoegtropIMC. However, because this PR would have an impact on the features available to the end-user, I suggest we delay it to 8.8.

**Comment by MSoegtropIMC** [comment]
I am still working on having an OCaml working properly on Windows. The issues are the same as in make, so I plan to fix make with it. It is making progress, but I am just very busy with my daily work since 3 months. As an intermediate solution the installation of make can be disabled. The top level make script has a command line option (-make=Y or -make=N) to build make or not. I would suggest to use this top level option if the delivery of make with the installer is not wanted.

In case this top level option (-make) doesn't work, please let me know and I will fix it. But the recommended way for removing the delivery of make is to use this documented command lien option of the build system.

Best regards,
Michael

File: dev/build/windows/makecoq_mingw.sh (hunk 1 above)
Comment (gares) [reply dev/build/windows/makecoq_mingw.sh]: According to Michael this code may be useful in the future, and I'd rather leave it there and eventually disable it via the command line (or build `make.exe`, but not ship it)

**Comment by maximedenes** [comment]
@MSoegtropIMC Do you have an opinion whether we should merge this, to improve the current situation (the wrong `make` can be picked), until your shell properly solves the issue?

**Comment by maximedenes** [comment]
@gares When you close a PR, can you add a message with summary of the reason? What was it, here?

**Comment by Zimmi48** [comment]
The reason was: superseded by #6881.

**Pull request closed without merging**